staubli_rx160_support/urdf/rx160_macro.xacro

```diff
@@ -117,42 +167,48 @@
       <parent link="${prefix}base_link"/>
       <child link="${prefix}link_1"/>
       <axis xyz="0 0 1"/>
-      <limit lower="-2.967060" upper="2.967060" effort="0" velocity="2.88" />
+      <limit lower="-2.967060" upper="2.967060" effort="692.0" velocity="2.88" />
```

Comment:
Could you point me to the doc/section/page where you got these `effort` limits from?